src/raftstore/store/worker/compact.rs

```diff
@@ -202,15 +203,15 @@ fn collect_ranges_need_compact(
     tombstones_num_threshold: u64,
     tombstones_percent_threshold: u64,
 ) -> Result<VecDeque<(Key, Key)>, Error> {
-    // Check the SST properties for each range, and we will compact a range if the range
+    // Checks the SST properties for each range, and we will compact a range if the range
     // contains too much RocksDB tombstones. we will merge multiple neighbouring ranges
```

Comment:
```suggestion
     // contains too many RocksDB tombstones. TiKV will merge multiple neighboring ranges
```

Reply:
Generally in PingCAP docs, we use American English (neighboring) rather than British English (neighbouring).

Reply:
@Hoverbear We must use a unified language form in our company and recommend using American English as our standard English.

Reply:
I know...  but...

![Okay. :(](https://media.giphy.com/media/3e18NPUVzoxzO/giphy-downsized.gif)